sources/engine/Stride.VirtualReality/OpenXR/OpenXRHmd.cs

```diff
@@ -348,6 +359,22 @@ public override unsafe void Enable(GraphicsDevice device, GraphicsDeviceManager
             Xr.StringToPath(Instance, "/user/hand/left", ref leftHandPath);
         }
 
+        public override IDisposable StartPassthrough()
+        {
+            if (!SupportsPassthrough)
+                throw new NotSupportedException();
```

Comment:
Can you add a message to the exception explaining "The device does not support Passthrough mode"?

Reply:
Done, also added a new exception in case passthrough mode wasn't requested at all.